com.unity.shadergraph/Documentation~/Refract-Node.md

```diff
@@ -0,0 +1,42 @@
+# Refract Node
+
+## Description
+
+Generates a refraction [Ibn Sahl-Snell-Descartes's Law](https://en.wikipedia.org/wiki/Snell%27s_law). Based on an **Incident** vector, the **Normal** of the surface and **Eta** we produce the refracted vector.
+
+Based on Ibn Sahl's Law, for a given refractive index of the medium, we can reach an angle where all the surface behave like a perfect mirror, that's why in **Safe** Mode generate a null vector, this implementation is here to not double the reflection intensity which is handle by Fresnel.
+
+## Ports
+
+| Name        | Direction           | Type  | Binding | Description |
+|:------------ |:-------------|:-----|:---|:---|
+| Incident      | Input | Vector | None | Incident normalied vector to the surface (From surface to source, for instance from pixel to eye). |
+| Normal      | Input | Vector | None | Normal normalized of the surface |
+| Eta      | Input | Float    | None | The ratio of refractive index (Eta == (source refractive index)/(medium refractive index)), For instance for an interaction from air to glass **Eta** will be 1.0/1.5 and for an interaction from glass to air **Eta** will be 1.5/1.0. |
```

Comment:
What are we referring to by the "source" refractive index here? The air around the item before it hits the medium of the shader? 

Also, is "Eta" the best term to use here? Is this supposed to be an acronym for something? If it is an acronym, it should be capitalized as ETA - but then I'd argue that there should be a better name that would avoid confusion with "estimated time of arrival".

Reply:
Eta is for the greek letter eta η which define the ratio from the source (where the light is coming from) and the medium IOR.
For example an Air-Glass Interation would have an eta of 0.666 (== 1.0/1.5)
And a Glass-Air Interaction would have an eta of 1.5 (== 1.5/1.0) with 1.0 as the IOR of Air and 1.5 the IOR of glass.

Reply:
> The refractive index ratio, which is defined by the Source Refractive Index (where the light is coming from) divided by the Medium Refractive Index (the medium causing the refraction). For example, the **Eta** for an air source and a glass medium would be 1.0/1/5, while a glass source to an air medium would be 1.5/1.0. 

Please change the text to the above.